### PR TITLE
Use date range format on offer modal

### PR DESCRIPTION
**Pull request by jrlarano**

### CODE REVIEW

**Comment by mortenbo** [comment]
@jrlarano Is it ready for review?

**Review by klarstrup — APPROVED** [review]

**Review by mortenbo — APPROVED** [review]

File: lib/kits/core-ui/components/common/offer-overview.ts [hunk 1]
```diff
@@ -206,6 +197,36 @@ const OfferOverview = ({
         addShoppingListListener();
     };
 
+    const formatOfferDate = (dateStr) => {
+        const date = parseDateStr(dateStr);
+        const month = `0${date.getUTCMonth() + 1}`.slice(-2);
+        const day = `0${date.getUTCDate()}`.slice(-2);
+
+        return `${day}/${month}`;
```
Comment (klarstrup) [inline lib/kits/core-ui/components/common/offer-overview.ts]: save some effort by using `date.toLocaleDateString(undefined, {month:"2-digit",day:"2-digit"})`

**Review by klarstrup — APPROVED** [review]
mergable but i have a tweak suggestion

File: lib/kits/core-ui/components/common/offer-overview.ts (hunk 1 above)
Comment (jrlarano) [reply lib/kits/core-ui/components/common/offer-overview.ts]: Cool! This is better :D 
Also, I tried adding the locale. Do you think it's fine @mortenbo ?
image

**Pull request merged**